### PR TITLE
Fix demo app crash

### PR DESCRIPTION
**Pull request by anshumansworld**
This includes fixes of large layout & demo app crash

### CODE REVIEW

File: library.layouts/src/main/java/net/pubnative/library/layouts/widget/LargeLayoutWidget.java [hunk 1]
```diff
@@ -299,7 +319,7 @@ public void onVASTPlayerPlaybackStart() {
     public void onVASTPlayerPlaybackFinish() {
 
         Log.v(TAG, "onVASTPlayerPlaybackFinish");
-        mPlayer.stop();
+        destroy();
```
Comment (cerberillo) [inline library.layouts/src/main/java/net/pubnative/library/layouts/widget/LargeLayoutWidget.java]: even if you destroy, if you don't stop, the player could continue working

Comment (anshumansworld) [inline library.layouts/src/main/java/net/pubnative/library/layouts/widget/LargeLayoutWidget.java]: but in `destroy()` we are doing `mPlayer.destroy();`. That would not be sufficient?

Comment (anshumansworld) [inline library.layouts/src/main/java/net/pubnative/library/layouts/widget/LargeLayoutWidget.java]: 👍  going to add `stop` if video is playing.

**Comment by anshumansworld** [comment]
[(1) BUG - Asset Group 15 & 20 - Demo App Crashes [1.5]](https://trello.com/c/cQqdTMML/1345-1-bug-asset-group-15-20-demo-app-crashes-1-5)

**Pull request merged**